keras/backend/tensorflow_backend.py

```diff
@@ -3346,28 +3346,41 @@ def bias_add(x, bias, data_format=None):
         Output tensor.
 
     # Raises
-        ValueError: In case of invalid `data_format` argument.
+        ValueError: In case of invalid `data_format` argument, or the input bias dimension is not expect
```

Comment:
Can you rephrase this? The meaning is unclear. Additionally, please make sure to introduce line breaks to keep line length reasonable.